runtime/autoload/health/provider.vim

```diff
@@ -410,28 +412,34 @@ function! s:check_python(version) abort
       call health#report_warn('Python 3.3+ is recommended.')
     endif
 
-    call health#report_info('Python'.a:version.' version: ' . pyversion)
+    call health#report_info('Python version: ' . pyversion)
     if s:is_bad_response(status)
-      call health#report_info(printf('%s-neovim version: %s (%s)', pyname, current, status))
+      call health#report_info(printf('pynvim version: %s (%s)', current, status))
     else
-      call health#report_info(printf('%s-neovim version: %s', pyname, current))
+      call health#report_info(printf('pynvim version: %s', current))
+      let [module_found, _msg] = provider#pythonx#CheckForModule(python_bin,
+            \ 'neovim', a:version)
+      if !module_found
+        call health#report_error('Importing "neovim" failed.',
+              \ [pip .' uninstall pynvim neovim',
+              \  pip .' install pynvim',
+              \  pip .' install neovim # only if needed by third-party software'])
```

Comment:
If this is part of a single "advice" you can use newlines instead of separate items.

Reply:
> Should the shell lines be prepended with $

I generally avoid that (unless it is really needed to avoid confusion) because it makes copy-paste more difficult.